src/stickApi.pas

```diff
@@ -0,0 +1,74 @@
+unit stickApi;
+
+interface
+
+uses
+ Classes,
+ Windows,
+ SysUtils,
+ multiplayer,
+ IdHTTP;
+
+const baseUrl = 'http://stickman.hu/api?mode=';    
+
+//TODO: remove
+type TAsync = class(TThread)
```

Comment:
Sajnos main unitba behuzva Classes (amibol TThread jon) hibat okoz mert nincs qualified nevfeloldas, es mas unittal conflictol. Lehet, hogy majd Delphi10 konverzio megoldja, ugyanakkor ott amugy is TTask amit hasznalni fogunk. Alternativan megmaradhat ez az api-related threadekhez, es kap egy privat TApi propertyt.